plugin-server/tests/shared/capabilities.test.ts

```diff
@@ -0,0 +1,45 @@
+import Piscina from '@posthog/piscina'
+
+import { KafkaQueue } from '../../src/main/ingestion-queues/kafka-queue'
+import { startQueues } from '../../src/main/ingestion-queues/queue'
+import { Hub, LogLevel } from '../../src/types'
+import { createHub } from '../../src/utils/db/hub'
+
+jest.mock('../../src/main/ingestion-queues/kafka-queue')
+
+describe('queue', () => {
```

Comment:
This is what I refactored `queue.test.ts` as that was just a tiny suite making sure the Redis queue works + an even smaller suite verifying that capability settings are used. I only kept the latter as the former is already covered very well for Kafka ingestion and I don't really see the value in adapting that exact test for Kafka.